packages/entity-detail/README.md

```diff
@@ -11,14 +11,16 @@ React-registry name: `entity-detail`
 | `entityName`           | x         | Entity name of the record
 | `entityId`             | x         | The ID of the entity 
 | `formName`             |           | Detail-form that should be loaded
+| `mode`                 |           | Determines if the detail is in create (`create`) or update (`update`) mode
```

Comment:
i would simplify the the description to "Determines if the detail is in create or update mode"